tests/testthat/test-util.R

```diff
@@ -491,11 +491,15 @@ test_that("show_question", {
 })
 
 test_that("show_question interactive", {
-  ## We can't check that show_question will return true on appveyor since
-  ## R CMD check is non-interactive
-  skip_on_cran()
-  skip_on_travis()
-  skip_on_appveyor()
-
-  expect_identical(show_question(), TRUE)
+  withr::with_options(
```

Comment:
unrelated, but this test was failing locally when run non-interactively and not being run remotely